plugins/masters-tournament/manager.py

```
@@ -171,6 +171,10 @@ def __init__(self, plugin_id, config, display_manager, cache_manager, plugin_man
 
     PHASE_MODES = {
         "off-season": [
+            # masters_countdown appears 3× out of 10 entries (~30% screen time)
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)